fog/sample-paykit/src/cached_tx_data/mod.rs

```diff
@@ -66,7 +65,15 @@ const TELEMETRY_NUM_TXOS_KEY: Key = telemetry_static_key!("num-txos");
 /// returned, and the client will not spend this TxOut, and the balance of this
 /// account will not reflect such TxOut's. This has to cover at least the
 /// default and change subaddress indexes.
-const SUBADDRESS_LOW_RANGE: RangeInclusive<u64> = 0..=DEFAULT_SUBADDRESS_INDEX;
+///
+/// Note: We also put the historical change subaddress of 1 in this range.
+/// This is because we ran the test client as a canary for several months using
+/// 1 as the change subaddress, so those accounts now have many subaddress index
+/// 1 TxOuts. Adding this to the searched range means that they don't lose all
+/// this money, and also silences the warnings when they get TxOuts from fog on
+/// "unexpected" subaddresses.
+const HISTORICAL_CHANGE_SUBADDRESS_INDEX: u64 = 1;
+const SUBADDRESS_LOW_RANGE: RangeInclusive<u64> = 0..=HISTORICAL_CHANGE_SUBADDRESS_INDEX;
```

Comment:
Why is this showing up in the diff?

Reply:
I cherry picked the PR on release/v2 into this one to master. I think this got pulled in from there. I'm not sure if this is still correct or not. The comment seems to make sense?

Reply:
It might be correct but it is unrelated to this PR :)

Reply:
Is there a reason to target master instead of release/v2?

I think we would release this in v2.1 if we can

Reply:
maybe its better to wait

Reply:
Seems to me that targeting `master` makes sense since we definitely need it in master, and only maybe we also want it in release/v2